src/vpp_plugins/vpp-chaperone/VppConfigManager.h

```diff
@@ -44,7 +44,7 @@ class VppConfigManager final
 
   // Add and delete CPE Policers.
   void doCpePolicerConfig (VppClient &vppClient, const std::string &interface,
-                           const folly::dynamic &policers);
+                           const folly::dynamic &policers, bool policerPresent);
```

Comment:
A better parameter name would be `isAdd` instead of `policerPresent` which is ambiguous

Reply:
Addressed Review Comment.